### PR TITLE
SEO-176062-Aurelia-controls-issues

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Aurelia/Schedule/how-to.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ api: /api/js/ejschedule
 
 The client-side validation of the fields present within the custom appointment window can be handled before submitting it, by adding appropriate validation classes to each field.
 
-Refer the steps [here](/aurelia/schedule/customization#appointment-window-customization) and create a sample for Custom Appointment window, before proceeding with the following validations.
+Refer the steps [here](https://help.syncfusion.com/aurelia/schedule/customization#appointment-window-customization) and create a sample for Custom Appointment window, before proceeding with the following validations.
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/how-to.md:16]: Why have you added the https link

File: Aurelia/Schedule/working-with-appointments.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ N> An all-day row is normally visible on the Scheduler, as the [showAllDayRow](/
 
 ### Recurrence
 
-Represents an appointment that is created for a certain time interval that occurs repeatedly in a daily, weekly, monthly, yearly or every weekday basis at the same time interval based on the recurrence rule. The other available options and validations that can be performed on recurrence appointments can be referred from [here](/aurelia/schedule/working-with-appointments#recurrence-options).
+Represents an appointment that is created for a certain time interval that occurs repeatedly in a daily, weekly, monthly, yearly or every weekday basis at the same time interval based on the recurrence rule. The other available options and validations that can be performed on recurrence appointments can be referred from [here](https://help.syncfusion.com/aurelia/schedule/working-with-appointments).
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:32]: Check this link

File: Aurelia/Schedule/working-with-appointments.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ The appointments can be added/edited in the Scheduler using any one of the follo
 * Quick window
 * Inline creation/editing
 * Default appointment window
-* [Context menu](/aurelia/schedule/context-menu)
+* [Context menu](https://help.syncfusion.com/aurelia/schedule/context-menu)
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:45]: Check this

File: Aurelia/Schedule/working-with-appointments.md [hunk 4]
```diff
@@ -227,10 +227,10 @@ The default appointment window is availed with options like
...
 The other additional options available are listed below for which the appropriate API’s are needed to be configured to display these options on the appointment window.
 
-* Location ([showLocationField](/aurelia/schedule/miscellaneous#showhide-location-field))
-* Priority ([prioritySettings](/aurelia/schedule/working-with-appointments#priority))
-* Categorize ([categorizeSettings](/aurelia/schedule/working-with-appointments#categorization))
-* [Resources](/aurelia/schedule/resources)
+* Location ([showLocationField](https://help.syncfusion.com/aurelia/schedule/miscellaneous#showhide-location-field))
+* Priority ([prioritySettings](https://help.syncfusion.com/aurelia/schedule/working-with-appointments#priority))
+* Categorize ([categorizeSettings](https://help.syncfusion.com/aurelia/schedule/working-with-appointments#categorization))
+* [Resources](https://help.syncfusion.com/aurelia/schedule/resources)
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:233]: Check all this links

File: Aurelia/Schedule/working-with-appointments.md [hunk 5]
```diff
@@ -335,7 +335,7 @@ The appointments can be deleted in either of the following ways,
 * Selecting an appointment and pressing <kbd>Delete</kbd> key.
 * Through Programmatically.
 
-A pop-up with a confirmation message will get displayed before deleting an appointment, which can be either switched on/off using the API `showDeleteConfirmationDialog`. Also, the confirmation text in that pop-up can be customized as mentioned [here](/aurelia/schedule/globalization-and-localization#localization:localizing-specific-words).
+A pop-up with a confirmation message will get displayed before deleting an appointment, which can be either switched on/off using the API `showDeleteConfirmationDialog`. Also, the confirmation text in that pop-up can be customized as mentioned [here](https://help.syncfusion.com/aurelia/schedule/globalization-and-localization#localization:localizing-specific-words).
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:338]: Check this link

File: Aurelia/Schedule/working-with-appointments.md [hunk 6]
```diff
@@ -390,7 +390,7 @@ export class Schedule {
 
 {% endhighlight %}
 
-N> All these CRUD operations on appointments (add/edit/delete) can also be done through the default [context menu](/aurelia/schedule/context-menu#default-menu-options) options **Add Appointment**, **Edit Appointment** and **Delete Appointment** which is available, when context menu settings is enabled within Scheduler.
+N> All these CRUD operations on appointments (add/edit/delete) can also be done through the default [context menu](https://help.syncfusion.com/aurelia/schedule/context-menu#default-menu-options) options **Add Appointment**, **Edit Appointment** and **Delete Appointment** which is available, when context menu settings is enabled within Scheduler.
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:393]: Check all this links that you've added https

File: Aurelia/Schedule/working-with-appointments.md [hunk 7]
```diff
@@ -1349,7 +1349,7 @@ It has the following properties based on which the recurrence appointments are r
     </tr>
 </table>
 
-To know more about other possible combinations of above specified recurrence rule properties, refer [here](http://www.syncfusion.com/kb/3719/what-is-recurrencerule-in-the-schedule-control).
+To know more about other possible combinations of above specified recurrence rule properties, refer [here](https://support.syncfusion.com/kb/article/3918/what-is-recurrencerule-in-the-schedule-control?isInternalRefresh=False).
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:1352]: Check this KB link

File: Aurelia/Schedule/working-with-appointments.md [hunk 8]
```diff
@@ -1431,7 +1431,7 @@ export class Schedule {
 
 {% endhighlight %}
 
-N> You can parse the **RecurrenceRule** of an appointment from the server-side by making use of a new generic utility class **RecurrenceHelper**. Refer this [KB document](https://www.syncfusion.com/kb/5390/how-to-parse-the-recurrencerule-in-server-side).
+N> You can parse the **RecurrenceRule** of an appointment from the server-side by making use of a new generic utility class **RecurrenceHelper**. Refer this [KB document](https://support.syncfusion.com/kb/article/5128/how-to-parse-the-recurrencerule-in-server-side-in-aspnet-mvc-scheduler?isInternalRefresh=False).
```
Comment (Yvone-Atieno) [inline Aurelia/Schedule/working-with-appointments.md:1434]: Check this link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Aurelia/Schedule/how-to.md (hunk 1 above)
Comment (Edithabich) [reply Aurelia/Schedule/how-to.md]: reverted

File: Aurelia/Schedule/working-with-appointments.md (hunk 2 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 3 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 4 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 5 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 6 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 8 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

File: Aurelia/Schedule/working-with-appointments.md (hunk 7 above)
Comment (Edithabich) [reply Aurelia/Schedule/working-with-appointments.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Fine.

**Pull request merged**